hr_holidays_notify_employee_manager/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "HR Holidays Notify Employee Manager",
+    "summary": "Notify employee's manager by mail on Leave Requests " "creation.",
+    "version": "13.0.1.0.0",
+    "category": "Human Resources",
+    "website": "https://github.com/OCA/hr",
```

Comment:
```suggestion
    "website": "https://github.com/OCA/hr-holidays",
```
Please change the website.